docs/README.md

```diff
@@ -1,18 +1,22 @@
 # Doctrine ORM Documentation
 
+The documentation is written in [ReStructured Text](https://docutils.sourceforge.io/rst.html).
+
 ## How to Generate:
-Using Ubuntu 14.04 LTS:
 
-1. Run ./bin/install-dependencies.sh
-2. Run ./bin/generate-docs.sh
+In the `docs/` folder, run
+
+    composer update
+
+Then compile the documentation with:
 
-It will generate the documentation into the build directory of the checkout.
+    make html
 
+This will generate the documentation into the `build` subdirectory.
 
-## Theme issues
+To browse the documentation, you need to run a webserver:
 
-If you get a "Theme error", check if the `en/_theme` subdirectory is empty,
-in which case you will need to run:
+    cd build/html
+    php -S localhost:8000
```

Comment:
i noticed that the file links are rendered as absolute paths rather than relative paths, making it impossible to browse the files without a webserver.

Reply:
This is currently not possible with the application like it is. The guides library supports this. But a lot of other things in this repo don't